src/utils/Select/Select.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import cn from 'classnames'
+import styles from './Select.css'
+  
+const Select = ({ options, selectClass }) => {
```

Comment:
You need to define props with **PropTypes**

Reply:
https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/c9362e737f87aed20a979cc23bf00d54e63dd86d